static/riotjs/wizard/grade_homework.tag

```diff
@@ -4,7 +4,10 @@
         <ol>
             <li each="{ question in definition.custom_questions }">{ question.question }
                 <ul>
-                    <li each="{ answer in question.student_answers }">{ answer }</li>
+                    <li if="{ question.question_type == 'UL' }" each="{ answer in question.student_answers }">
+                        <a href="{ answer }" target="_blank">{ answer }</a>
+                    </li>
+                    <li if="{ question.question_type != 'UL' }" each="{ answer in question.student_answers }">{ answer }</li>
                 </ul>
```

Comment:
I am not seeing the student's answers to these questions in the grading view here. Let me know if you see the same behavior. I'm experiencing the same on the submission detail page. I can only see questions, and not the responses.

Reply:
Also, the `homework_answers.html` page should display these as links. Right now, I don't believe I can see the answer.

Reply:
Bug fixed good catch.